cookbook/controller/error_pages.rst

```diff
@@ -46,7 +46,7 @@ is used to render a Twig template to show the user.
 
 .. _cookbook-error-pages-by-status-code:
 
-This controller uses the HTTP status code, request format and the following
+This controller uses the HTTP status code, the request format and the following
```

Comment:
not sure if this is needed. For me (but maybe because I'm dutch), the original sentence reads easier.